lib/interface/i_my_repository.dart

```diff
@@ -0,0 +1,6 @@
+import '../shared/movie.dart';
+
+abstract class IMyRepository {
+  Future<List<Movie>> loadMovies();
+  Future<List<dynamic>> loadGenres();
```

Comment:
Why a `List<dynamic>`? Should be `List<Genre>`

Reply:
fixed